<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRowDialog</class>
 <widget class="QDialog" name="AddRowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new row</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="8">
    <widget class="QLabel" name="ageLabel_10">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8">
    <widget class="QLabel" name="ageLabel_2">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="ageLabel_9">
     <property name="text">
      <string>Color 5</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="LineEditColor2">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>#ffffff</string>
     </property>
    </widget>
   </item>
   <item row="6" column="8">
    <widget class="QLabel" name="ageLabel_7">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="FileLabel_2">
     <property name="text">
      <string>.jpg</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>239</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Percantage</string>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <widget class="QLabel" name="ageLabel_8">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="FileLabel">
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
   </item>
   <item row="4" column="8">
    <widget class="QLabel" name="ageLabel_4">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QLineEdit" name="LineEditPers3">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="LineEditColor3">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>#ffffff</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QLineEdit" name="LineEditPers4">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="LineEditColor1">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>#ffffff</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="ageLabel_3">
     <property name="text">
      <string>Color 2</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="LineEditColor4">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>#ffffff</string>
     </property>
    </widget>
   </item>
   <item row="17" column="7">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="ageLabel_5">
     <property name="text">
      <string>Color 3</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ageLabel">
     <property name="text">
      <string>Color 1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QLineEdit" name="LineEditPers2">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="6">
    <widget class="QLineEdit" name="FileLineEdit">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>IMG_000</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="ageLabel_6">
     <property name="text">
      <string>Color 4</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="LineEditColor5">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>#ffffff</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLineEdit" name="LineEditPers1">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="7">
    <widget class="QLineEdit" name="LineEditPers5">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Colors (hex)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
